contracts/ownable.sol

```diff
@@ -6,6 +6,7 @@ pragma solidity ^0.5.0;
 */
 contract Ownable {
   address private _owner;
+  address payable _ownerPayable;
```

Comment:
Beginning 0.5.0 the address type was split into address and address payable, where only address payable provides the transfer function.